enh(settings): Load from disabled users endpoint

Pytal opened the pull request:
- Use endpoint https://github.com/nextcloud/server/pull/39756

## Summary

- Load users directly from disabled users endpoint

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

Pytal committed:
enh(settings): Load from disabled users endpoint

Signed-off-by: Christopher Ng <chrng8@gmail.com>

come-nc reviewed (changes requested):
User count seems broken, also avatar of freshly disabled users do not load.

come-nc commented on apps/settings/src/components/UserList.vue:
```diff
@@ -314,6 +314,10 @@ export default {
 		},
 
 		async handleScrollEnd() {
+			// Do not load more users when all have already been loaded
+			if (this.users.length % 25 !== 0) {
+				return
+			}
```
Hum, what if I load the list, enable a user from the list, and then scroll to the end? Length will be 24, no?

Pytal replied:
Nope, only `filteredUsers` would change while `users` which contains all users loaded would stay the same

come-nc commented:
It works for disabled user list but for active user list it does not load the next ones when scrolling to the end it seems @Pytal

Pytal committed:
chore: compile assets

Signed-off-by: Christopher Ng <chrng8@gmail.com>

come-nc committed:
Invert parameter order in getDisabledUserList to be consistent

This matches what was done in the calls and so fixes getting disabled
 user list when there are several backends returning disabled users.

Signed-off-by: Côme Chilliet <come.chilliet@nextcloud.com>

come-nc commented:
This is still not working correctly for me for active user list.
The problem is that with LDAP backend it returns more users than the limit, because the limit is applied to each backend indivdually. This may be seen as a bug but has been this way for a long time. (and fixing that brings perf questions)

So the test that user count is a multiple of 25 does not work, in may case the first request with limit 25 and offset 0 returns 37 users (25 from ldap, the rest from DB).
What you need to test to know if there are no more users is if the number of returned users is less than the limit.

Note that for disabled users endpoint it behaves better and will slice to return only 25 users when limit is 25, but maybe it’s still safer to consider 25 users **or more** valid sizes for non-complete answer.
